spec/std/isa/inst/Q/fmvp.q.x.yaml

```
@@ -6,9 +6,16 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: fmvp.q.x
-long_name: No synopsis available
-description: |
-  No description available.
+long_name: Floating-Point Move Integer Pair Quad-Precision
```

```suggestion
long_name: Floating-Point Move to Quad-Precision from Integer Pair
```
(some of these don't read well left-to-right, do they?)